n3fit/src/evolven3fit_new/eko_utils.py

```diff
@@ -0,0 +1,78 @@
+from ekobox import gen_theory, gen_op
+from eko import run_dglap
+
+from validphys.api import API 
+from . import utils
+
+from typing import Any, Dict, Optional
+import logging
+
+_logger = logging.getLogger(__name__)
+
+def construct_eko_cards(theoryID, q_fin, q_points, x_grid, op_card_dict: Optional[Dict[str, Any]] = None, t_card_dict: Optional[Dict[str, Any]] = None):
```

Comment:
Please make this line shorter. 

As a matter of taste, I find the type annotations are not especially helpful in this case. E.g. `op_card_dict=None` already communicates pretty much the same as `Optional[Dict[str, Any]]`.